go-controller/pkg/factory/factory.go

```diff
@@ -394,7 +397,31 @@ func getObjectMeta(objType reflect.Type, obj interface{}) (*metav1.ObjectMeta, e
 
 type AddHandlerFuncType func(namespace string, sel labels.Selector, funcs cache.ResourceEventHandler, processExisting func([]interface{}) error) (*Handler, error)
 
+func (wf *WatchFactory) GetHandlerPriority(objType reflect.Type) (priority uint32) {
```

Comment:
i think you need comments here explaining why the priorities are set to what they are for each type

Reply:
done.